sql/telemetry_derived/clients_first_seen_v1/init.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TEMPORARY TABLE clients_first_seen_dates
+PARTITION BY
+  first_seen_date
+AS
+WITH base AS (
+  SELECT
+    client_id,
+    ARRAY_AGG(submission_date) AS dates_seen,
+  FROM
+    clients_daily_v6
```

Comment:
It looks like for most queries in this repository the dataset is usually specified. So maybe `telemetry_derived.clients_daily_v6`?

Reply:
Looks like it's been fixed.